lib/grpmat.gi

```diff
@@ -538,6 +559,27 @@ local map;
   fi;
 end);
 
+InstallMethod( IsomorphismPermGroup,
+    "finite matrix group",
+    [ IsMatrixGroup and IsFinite ], 5-NICE_FLAGS,
+    function( G )
+    local map;
```

Comment:
Instead of duplicating this code via copy&paste, how about instead moving storing it as its own function (`BindGlobal("IsomorphismPermGroupForMatrixGroup", function ... end);`) and then we can call `InstallMethod` twice for it?

Reply:
Yes. I had sketched this in an earlier comment.